internal/product/product.go

```diff
@@ -54,8 +53,8 @@ type UOM struct {
 }
 
 type ProductVendor struct {
-	Product
-	vendors.Vendor
+	ProductID string `db:"product_id" json:"product_id"`
+	VendorID  string `db:"vendor_id" json:"vendor_id"`
```

Comment:
Is there any reason rewriting this as VendorID and ProductID, instead of the object?

Reply:
I don't see any use case that needed both the object. Additionally, I just need to store both the ID and the data type name is relevant, matching the table.  Later, if we need to retrieve both objects after doing sql query, I believe would be better to create a new custom return data type for that.

Reply:
I see, got it! Thanks